nebd/src/part2/request_executor_curve.cpp

```diff
@@ -34,50 +34,53 @@ using ::curve::client::UserInfo_t;
 const char* kSessionAttrKey = "session";
 const char* kOpenFlagsAttrKey = "openflags";
 
-curve::client::OpenFlags ConverToCurveOpenFlags(const OpenFlags* flags) {
+curve::client::OpenFlags ConverToCurveOpenFlags(const OpenFlags* flags, std::string confPath) {
     curve::client::OpenFlags curveflags;
-
+    curveflags.confPath = confPath;
+    
     if (!flags) {
         return curveflags;
     } else {
         if (flags->has_exclusive()) {
             curveflags.exclusive = flags->exclusive();
         }
-
         return curveflags;
     }
 }
 
-std::string FileNameParser::Parse(const std::string& fileName) {
+std::pair<std::string, std::string> FileNameParser::Parse(const std::string& fileName) {
+    std::string confPath = "";
```

Comment:
you don't have to init it from a empty string, it's empty by default.

Reply:
done